jkube-kit/profile/src/test/java/org/eclipse/jkube/kit/profile/ProfileUtilTest.java

```diff
@@ -32,29 +32,6 @@
  */
 class ProfileUtilTest {
 
-    @Test
-    void simple() throws IOException {
-        InputStream is = getClass().getResourceAsStream("/jkube/config/profiles-lookup-dir/profiles.yaml");
-        assertThat(is).isNotNull();
-        List<Profile> profiles = ProfileUtil.fromYaml(is);
-        assertThat(profiles).isNotNull().hasSize(4)
-            .first().hasFieldOrPropertyWithValue("name", "simple")
-            .satisfies(profile -> assertThat(profile.getEnricherConfig())
-                .returns(true, c -> c.use("base"))
-                .returns(false, c -> c.use("blub")))
-            .satisfies(profile -> assertThat(profile.getGeneratorConfig())
-                .returns(false, c -> c.use("java.app"))
-                .returns(true, c -> c.use("spring.swarm")));
-    }
-
-    @Test
-    void multiple() throws IOException {
-        InputStream is = getClass().getResourceAsStream("/jkube/config/ProfileUtilTest-multiple.yml");
-        assertThat(is).isNotNull();
-        List<Profile> profiles = ProfileUtil.fromYaml(is);
-        assertThat(profiles).hasSize(2);
-    }
```

Comment:
Why are these tests deleted? How is this change related to KubernetesClient upgrade?

Reply:
okay, I see you're removed ProfileUtil.fromYaml method

Reply:
Some of the tests were redundant and no longer applicable.

The remaining tests should cover all of the scenarios